mongoDB/index.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+const mongoose = require('mongoose');
+const { pictures, reviewPhotos } = require('./sampleData');
+
+mongoose.connect('mongodb://localhost/pictures', { useUnifiedTopology: true, useNewUrlParser: true });
+
+const db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+});
+
+const PicturesSchema = mongoose.Schema({
+  item_id: { type: Number, unique: true },
+  store_id: Number,
+  item_pictures: [{ large: String, normal: String, thumbnail: String }],
+  seller_picture: String,
+  store_picture: String,
+});
+
```

Comment:
Is there a reason why you didn't use a models folder for your schema?  I have only seen it done in a models folder - so that's why I'm asking.